pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -192,6 +194,10 @@ def work(self):
                                 if status is 1:
 
                                     id_list2 = self.count_pokemon_inventory()
+                                    pokemon_to_transfer = list(Set(id_list2) - Set(id_list1))
+                                    if len(pokemon_to_transfer) == 0:
+                                        raise RuntimeError(
```

Comment:
Maybe it's because I don't have a lot of state on this, but why are we raising a runtime error?  Shouldn't the bot just "return" in this case?

I don't think workers should cause the bot to exit